detect_variants/add_bam_readcount_to_vcf.cwl

```diff
@@ -4,32 +4,26 @@ cwlVersion: v1.0
 class: CommandLineTool
 label: "add bam_readcount info to vcf"
 
-baseCommand: ["/usr/bin/python3", "/usr/bin/add_bam_readcount_to_vcf_helper.py"]
+baseCommand: ["vcf-readcount-annotator"]
```

Comment:
We've often been specifying the full path to these rather than trusting they're in the `$PATH`.

Reply:
It's a pip package. Not sure where that would get installed to in our setup.

Reply:
We could try `which vcf-readcount-annotator` inside the container to find the full path.  The thought behind my comment is to make sure it works when using the scrubbed user environment, so if that's still functioning we needn't worry :smile:

Reply:
I ran the exome test as described here https://confluence.ris.wustl.edu/pages/viewpage.action?spaceKey=CI&title=Toil#Toil-PipelineTest. Not sure if that is sufficient.

Reply:
That ought to do fine!  Hopefully we'll eventually set up an automated test for that.